utils/node_local_test.py

```diff
@@ -1300,6 +1312,163 @@ def run_in_fg(server, conf):
+        pools = make_pool(server)
+
+    pool = pools[0]
+
+    container = str(uuid.uuid4())
+
+    dfuse = DFuse(server, conf, pool=pool)
+
+    print('Creating container and populating')
+    count = 4
+    dfuse.start()
+    parent = os.path.join(dfuse.dir, container)
+    try:
+        os.mkdir(parent)
+    except FileExistsError:
+        pass
+    create_time = make_dirs(parent, count)
+    dfuse.stop()
+
+    results = OrderedDict()
+    results[count] = OrderedDict()
+    results[count]['create_time'] = create_time
+    all_start = time.time()
+
+    while True:
+
+        dfuse = DFuse(server, conf, pool=pool, container=container)
+        dir_dir = os.path.join(dfuse.dir,
+                               'dirs.{}'.format(count))
+        file_dir = os.path.join(dfuse.dir,
+                                'files.{}'.format(count))
+        dfuse.start()
+        start = time.time()
+        subprocess.run(['/bin/ls', '-U', dir_dir], stdout=subprocess.PIPE)
+        elapsed = time.time() - start
+        print('processed {} dirs in {:.2f} seconds'.format(count,
+                                                           elapsed))
+        results[count]['dirs'] = elapsed
+        dfuse.stop()
+        dfuse = DFuse(server, conf, pool=pool, container=container)
+        dfuse.start()
+        start = time.time()
+        subprocess.run(['/bin/ls', '-U', file_dir], stdout=subprocess.PIPE)
+        elapsed = time.time() - start
+        print('processed {} files in {:.2f} seconds'.format(count,
+                                                            elapsed))
+        results[count]['files'] = elapsed
+        dfuse.stop()
+
+        dfuse = DFuse(server, conf, pool=pool, container=container)
+        dfuse.start()
+        start = time.time()
+        subprocess.run(['/bin/ls', '-t', dir_dir], stdout=subprocess.PIPE)
+        elapsed = time.time() - start
+        print('processed {} dirs in {:.2f} seconds'.format(count,
+                                                           elapsed))
+        results[count]['dirs_with_stat'] = elapsed
+        dfuse.stop()
+        dfuse = DFuse(server, conf, pool=pool, container=container)
+        dfuse.start()
+        start = time.time()
+        subprocess.run(['/bin/ls', '-t', file_dir], stdout=subprocess.PIPE)
+        elapsed = time.time() - start
+        print('processed {} files in {:.2f} seconds'.format(count,
+                                                            elapsed))
+        results[count]['files_with_stat'] = elapsed
+
+        elapsed = time.time() - all_start
+        if elapsed > 5 * 60:
+            dfuse.stop()
+            break
+
+        print_results(results)
+        count *= 2
+        results[count] = OrderedDict()
+        create_time = make_dirs(dfuse.dir, count)
+        results[count]['create_time'] = create_time
+        dfuse.stop()
+
+    run_daos_cmd(conf, ['container',
+                        'destroy',
+                        '--svc', '0',
```

Comment:
no need to specify --svc argument. it will be removed in the near future too. Recommend removing its use here in this patch.

Reply:
True, I've got another patch in flight which removes all the svc options from here already but I'll take this out in a respin.